clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CloudConfigRefreshConfig.java

```diff
@@ -16,37 +16,42 @@
 
 package com.netflix.spinnaker.clouddriver.cache;
 
-import com.netflix.spinnaker.cats.cluster.DefaultAgentIntervalProvider;
+import com.netflix.spinnaker.clouddriver.config.CloudConfigRefreshProperties;
 import com.netflix.spinnaker.clouddriver.refresh.CloudConfigRefreshScheduler;
 import com.netflix.spinnaker.kork.configserver.autoconfig.RemoteConfigSourceConfigured;
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.cloud.config.server.EnableConfigServer;
 import org.springframework.cloud.context.refresh.ContextRefresher;
 import org.springframework.context.annotation.*;
 
 /**
  * Create a {@link CloudConfigRefreshScheduler} to refresh the Spring Cloud Config Server from an
- * environment repository backend on a schedule that matches the cache refresh schedule.
+ * environment repository backend on a schedule that can be defined in the spring profile or has a
+ * sensible default (60 seconds)..
  */
 @Configuration
 @AutoConfigureAfter({RedisCacheConfig.class})
```

Comment:
This `@AutoConfigureAfter` can be removed now that there is no relationship to the cache config.

Reply:
Removed, thanks!